client/jsx/lib/search.jsx

```diff
@@ -229,7 +229,7 @@ function searchCallback(data, textStatus) {
 
     if (data.length > 0) {
 
-	$('#txt_subtitle').html(data[0]);
+	$('#txt_subtitle').html(escape(data[0]));
```

Comment:
This part of this file isn't really doing anything.  The state of the current search page is complicated, and I can explain more in person.